src/View/Helper/FormHelper.php

```diff
@@ -359,12 +359,15 @@ public function create($model = null, array $options = [])
             'encoding' => strtolower(Configure::read('App.encoding')),
             'templates' => null,
             'idPrefix' => null,
+            'valuesSources' => [],
```

Comment:
Right my mind was broken when I removed it! 'Context' alone would do it. Both or just context?